docs/pages/database-access/reference/cli.mdx

```diff
@@ -327,8 +329,10 @@ $ tsh db connect --db-user=alice --db-name=db example
 
 | Flag | Description |
 | - | - |
-| `--db-user` | Optionally, set database user name to connect as. |
-| `--db-name` | Optionally, set database name to connect to. |
+| `--db-user` | The database user to connect as. |
+| `--db-name` | The database name to connect to. |
```

Comment:
Let's use language in both places. I don't care if we say "to log in as" or "to connect as" but we should pick one and be consistent.